lib/matplotlib/projections/geo.py

```diff
@@ -52,8 +53,8 @@ def cla(self):
 
         self.grid(rcParams['axes.grid'])
 
-        Axes.set_xlim(self, -np.pi, np.pi)
-        Axes.set_ylim(self, -np.pi / 2.0, np.pi / 2.0)
+        Axes.set_xlim(self, -math.pi, math.pi)
```

Comment:
You can just use np.pi in most of these places, because set_xlim/etc. will directly convert everything to numpy scalars anyways, obviating any speedup.  (Using python scalars is only useful if you do some computations with them, and even then, probably the gain of writing `math.pi*2` below is obviated by the additional builtin->numpy conversion.)